source/state_representation/README.md

```diff
@@ -333,16 +337,22 @@ Those operations are very useful to convert `JointState` from `CartesianState` a
 The simplest conversion is to transform a `JointVelocities` into a `CartesiantTwist` by multiplication with the `Jacobian`
 
 ```cpp
+state_representation::Jacobian jac("myrobot", 3, "eef_frame", "base_frame");
 state_representation::JointVelocities jv("myrobot", 3);
-state_representation::CartesianTwist eef_twist = jac * js;
+// compute the twist of eef_frame in base_frame from the joint velocities
+state_representation::CartesianTwist eef_twist = jac * jv;
 ```
 
 The opposite transformation, from `CartesianTwist` to `JointVelocities` requires the multiplication with the `inverse`
 (or `pseudoinverse`).
 
 ```cpp
+state_representation::Jacobian jac("myrobot", 3, "eef");
 state_representation::CartesianTwist eef_twist("eef")
 state_representation::JointVelocities jv = jac.pseudoinverse() * eef_twist;
+// in case of non matching frame or reference frame throw an IncompatibleStatesException
+state_representation::CartesianTwist eef_link("link2", "link0")
```

Comment:
should this be `link_twist` instead of `eef_link`?

Reply:
oh yeah good catch